src/modm/platform/core/cortex/startup.c.in

```diff
@@ -102,10 +106,14 @@ void __modm_startup(void)
 	SCB_EnableICache();
 %% endif
 
-%% if "m0" not in core
+%% if has_vtor
 	// Set the vector table location
 	SCB->VTOR = (uint32_t)__vector_table_{{ vector_table_location }}_start;
+%% else
+	modm_platform_vector_table_remap();
+%% endif
```

Comment:
I don't like this, it's vendor specific. Should be done in `__modm_initialize_platform()`. Leave startup alone!1!!